calculator.c

```diff
@@ -0,0 +1,44 @@
+/* Christopher Liu - First take input of chosen operation, then take inputs of numbers. Then operate on them. Mod converts to ints first. */
```

Comment:
Hi Christopher - your code for this problem didn't compile. You forgot the #include <stdio.h> at the beginning and forgot to put curly braces after the switch statement.